spec.html

```diff
@@ -27723,15 +27728,15 @@ <h1>Date Time String Format</h1>
         <!-- es6num="20.3.1.16.1" -->
         <emu-clause id="sec-extended-years">
           <h1>Extended Years</h1>
-          <p>ECMAScript requires the ability to specify 6 digit years (extended years); approximately 285,426 years, either forward or backward, from 01 January, 1970 UTC. To represent years before 0 or after 9999, ISO 8601 permits the expansion of the year representation, but only by prior agreement between the sender and the receiver. In the simplified ECMAScript format such an expanded year representation shall have 2 extra year digits and is always prefixed with a + or - sign. The year 0 is considered positive and hence prefixed with a + sign.</p>
+          <p>ECMAScript requires the ability to specify 6 digit years, or <dfn>extended years</dfn>, to render the full range of a time value; -100,000,000 days to 100,000,000 days, or approximately -273,790 to 273,790 years, relative to midnight at the beginning of 01 January, 1970 UTC (<emu-xref href="#sec-time-values-and-time-range"></emu-xref>). To represent years before 0 or after 9999, ISO 8601 permits the expansion of the year representation, but only by prior agreement between the sender and the receiver. In the simplified ECMAScript format such an expanded year representation shall have 2 extra year digits and is always prefixed with a + or - sign. The year 0 is considered positive and hence prefixed with a + sign. ISO 8601 formatted strings with extended years representing points in time outside the range of a time value are not accepted.</p>
```

Comment:
I'm not sure if "ISO 8601 formatted strings with extended years representing points in time outside the range of a time value are not accepted." makes it clear that these date/time strings will result in an Invalid Date rather than being allowed to fall back to the implementation-specific parser. I think it'd be best if this behavior were explicitly described in `Date.parse`. (Apologies for not mentioning this the first time around.)

Reply:
The new text

> ISO 8601 formatted strings with extended years representing points in time outside the range of a time value are represented as the Number *NaN* when parsed by `Date.parse()` and do not fall back to implementation-specific behavior or heuristics.

Looks great to me. Thanks for the clarification.